src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,11 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+    while (true) {
+        println("Please enter a name (or press Enter to quit)")
+        var username : String? = readlnOrNull()
+        if (username == null || username == "") return else println("Hello, $username")
```

Comment:
Please check previous comment about kotlin's string check built in functions, or check if your changes were pushed